portality/api/current/crud/applications.py

```diff
@@ -197,7 +197,7 @@ def retrieve(cls, id, account):
 
         # is the current account the owner of the application
         # if not we raise a 404 because that id does not exist for that user account.
-        if ap.owner != account.id:
+        if account.is_super not in account.role and ap.owner != account.id:
```

Comment:
I don't think you tested this before committing it.

is_super is a function which determines if the account is the super user.

```
if not account.is_super and ap.owner != account.id
```

Reply:
> I don't think you tested this before committing it.

I'm afraid you're right. I wanted to push it quickly but got distracted. I'll fix it asap. I'm sorry for that!